mapper/census.go

```diff
@@ -361,6 +364,18 @@ func CreateCensusDatasetLandingPage(isEnableMultivariate bool, ctx context.Conte
 		p.DatasetLandingPage.QualityStatements[qsLen-1].CssClasses = append(p.DatasetLandingPage.QualityStatements[qsLen-1].CssClasses, "ons-u-mb-l")
 	}
 
+	if isFilterOutput {
+		analyticsJavaScript, err := getDataLayerJavaScript(getFilterAnalytics(fDims))
```

Comment:
What about if there is an error? As you're not handling it, presuming that you don't want the server to error at all? If that's the case, I'd suggest the following
```suggestion
		analyticsJavaScript := getDataLayerJavaScript(getFilterAnalytics(fDims))
```